### PR TITLE
Clean configs

### PR DESCRIPTION
**Pull request by jessegoodier**
Fixing a few issues, please review carefully. 

Added this:
https://guide.kubecost.com/hc/en-us/articles/6446286863383-Tuning-Resource-Consumption#on-secondaries-disabling-cloud-assets-and-running-kubecost-in-agent-mode-with-etl-and-caching-disabled

### CODE REVIEW

**Comment by kc-creynold** [comment]
@jessegoodier not a clickable link:

Learn more about how to configure this here: https://github.com/kubecost/docs/blob/main/cloud-integration.md#cloud-assets

**Comment by jessegoodier** [comment]
> @jessegoodier not a clickable link:
> 
> Learn more about how to configure this here: https://github.com/kubecost/docs/blob/main/cloud-integration.md#cloud-assets

@kc-creynold that's a seperate doc that I got some of the recomendations from. Will fix it though.

File: azure/values-azure-secondary.yaml [hunk 1]
```diff
@@ -9,26 +10,28 @@ kubecostModel:
...
-    enabled: false
 prometheus:
   server:
     global:
       external_labels:
+        # cluster_id should be unique for all clusters and the same value as .kubecostProductConfigs.clusterName
         cluster_id: kubecostProductConfigs_clusterName
     retention: 2d
+  alertmanager:
+    enabled: false
```
Comment (srpomeroy) [inline azure/values-azure-secondary.yaml:27]: I think we can safely just remove the alert manager block as it's disabled by default.

File: gcp/values-google-secondary.yaml [hunk 2]
```diff
@@ -7,26 +8,28 @@ kubecostModel:
...
-    enabled: false
 prometheus:
   server:
     global:
       external_labels:
+        # cluster_id should be unique for all clusters and the same value as .kubecostProductConfigs.clusterName
         cluster_id: kubecostProductConfigs_clusterName
     retention: 2d
+  alertmanager:
+    enabled: false
```
Comment (srpomeroy) [inline gcp/values-google-secondary.yaml:25]: I think we can safely just remove the alert manager block as it's disabled by default.

**Review by srpomeroy — APPROVED** [review]
LGTM, two potential tweaks but not neccesary.

**Pull request merged**